### PR TITLE
Fix password logging

### PR DESCRIPTION
**Pull request by v-ahibr**
FIPS property trustStorePassword was being logged

### CODE REVIEW

File: src/main/java/com/microsoft/sqlserver/jdbc/Util.java [hunk 1]
```diff
@@ -373,9 +373,14 @@ else if (ch == ':')
                         name = SQLServerDriver.getNormalizedPropertyName(name, logger);
                         if (null != name) {
                             if (logger.isLoggable(Level.FINE)) {
-                                if ((false == name.equals(SQLServerDriverStringProperty.USER.toString()))
-                                        && (false == name.equals(SQLServerDriverStringProperty.PASSWORD.toString())))
```
Comment (v-xiangs) [inline src/main/java/com/microsoft/sqlserver/jdbc/Util.java]: just for my own curiosity, why removed `&& (false == name.equals(SQLServerDriverStringProperty.PASSWORD.toString()))` ?

File: src/main/java/com/microsoft/sqlserver/jdbc/Util.java [hunk 2]
```diff
@@ -373,9 +373,14 @@ else if (ch == ':')
...
-                                    logger.fine("Property:" + name + " Value:" + value);
+                                if (false == name.equals(SQLServerDriverStringProperty.USER.toString())) {
+                                    if (!name.contains("password") && !name.contains("Password")) {
```
Comment (v-xiangs) [inline src/main/java/com/microsoft/sqlserver/jdbc/Util.java]: maybe we can also use `name.toLowerCase().contains("password")`

Comment (v-ahibr) [inline src/main/java/com/microsoft/sqlserver/jdbc/Util.java]: either works.
only reason i wrote it this way is that a similar if condition is used for  SQLServerDataSource

```
private void setStringProperty(Properties props,
            String propKey,
            String propValue) {
        if (loggerExternal.isLoggable(java.util.logging.Level.FINER) && !propKey.contains("password") && !propKey.contains("Password")) {
            loggerExternal.entering(getClassNameLogging(), "set" + propKey, propValue);
        }
        else
            loggerExternal.entering(getClassNameLogging(), "set" + propKey);
        if (null != propValue)
            props.setProperty(propKey, propValue);
        loggerExternal.exiting(getClassNameLogging(), "set" + propKey);
    }
```

File: src/main/java/com/microsoft/sqlserver/jdbc/Util.java (hunk 1 above)
Comment (v-ahibr) [inline src/main/java/com/microsoft/sqlserver/jdbc/Util.java]: Originally, the condition checked if we are working with the database password and then decides not to log it.

Now, we check if the property name includes the string "password" so the first if condition is not needed anymore

**Pull request closed without merging**